Fix slow sub desc (#1036)
emqtt/emq-docs-cn: en_US/advanced/alarms.md

| alarm.validity_period         | duration | 24h           | The maximum storage time of deactivated alarms, and expired alarms will be cleared |

{% emqxee %}

> EMQX Enterprise raises an alarm when the license expiration date is less than 30 days or if the number of connections exceeds the high water mark. The user can adjust the high/low water mark for the number of connections according to the actual situation.

Reviewer: ```suggestion
EMQX Enterprise raises an alarm when the license expirates in less than 30 days, or if the number of connections exceeds the high watermark. User can adjust the high/low watermark for the number of connections according to the actual situation.
```
